python/pyarrow/tests/test_flight.py

```diff
@@ -506,19 +508,16 @@ def test_flight_do_get_ticket():
     """Make sure Tickets get passed to the server."""
     data1 = [pa.array([-10, -5, 0, 5, 10], type=pa.int32())]
     table = pa.Table.from_arrays(data1, names=['a'])
-    with flight_server(
-            CheckTicketFlightServer,
-            expected_ticket=b'the-ticket',
-    ) as server_location:
-        client = flight.FlightClient.connect(server_location)
+    with CheckTicketFlightServer(expected_ticket=b'the-ticket') as server:
+        client = flight.FlightClient.connect(server.location)
```

Comment:
If `FlightClient.connect` is deprecated, then it shouldn't be used in the tests IMHO.

Reply:
Done.